index.js

```diff
@@ -5,10 +5,12 @@ module.exports = function(source) {
   this.cacheable && this.cacheable(true);
 
   var query = loaderUtils.parseQuery(this.query);
-  var tree = [
-    "require('nativejsx/dist/nativejsx-prototypes.js');",
-    nativejsx.transpile(source, query)
-  ];
+  var tree = [];
+
+  if (query && (query.prototypes !== 'inline')) {
```

Comment:
Mind checking for `'module'` as well. We shouldn't include prototypes in that case.